### PR TITLE
fix commons dependency conflict

### PR DESCRIPTION
**Pull request by z7658329**
fix commons dependency conflicts and prevent future uncertainty issues.
also see: [445](https://github.com/apache/incubator-hugegraph-toolchain/issues/445)

### CODE REVIEW

**Review by javeme — DISMISSED** [review]

File: pom.xml [hunk 1]
```diff
@@ -89,7 +89,7 @@
     </scm>
 
     <properties>
-        <revision>1.0.0</revision>
+        <revision>1.0.1</revision>
```
Comment (imbajin) [inline pom.xml:92]: search CommonVersion class & also need update it 

btw, we need a better way to modify the num, at least add a comment in pom?

Comment (imbajin) [reply pom.xml:92]: > If it is packaged in a jar, we cannot get the version from the pom.xml, so it fallback to the default version.

OK, we could consider these way: (way2 is the easiest)
1. check if somewhere possible for us to put the `version` into the `jar` file (like `meta` file/info of a jar?) 
2. just leave a comment before `revision` param in pom file (devs could see the `NOTE`)
3. remove the common version check, seems it's not necessary to check it?

Comment (z7658329) [reply pom.xml:93]: > 

also need to update RpcVersion.java

**Review by imbajin — APPROVED** [review]

**Comment by z7658329** [comment]
image

CodeQL / Analyze (java) (pull_request)  Step  always failed cause timeout exception (tried 4 times), is there any way to avoid such problems？ 
@javeme @imbajin

**Comment by imbajin** [comment]
> image
> 
> CodeQL / Analyze (java) (pull_request) Step always failed cause timeout exception (tried 4 times), is there any way to avoid such problems？ @javeme @imbajin

1. we could add a `settings.xml` for it to download the dependencies from different repo source
2. find a action to rerun it automatically (like auto retry 5 times if failed..)

File: pom.xml (hunk 1 above)
Comment (javeme) [reply pom.xml:93]: let's keep the status quo until we find a perfect way

**Review by javeme — APPROVED** [review]

**Comment by z7658329** [comment]
image

always build error

**Comment by z7658329** [comment]
> image
> 
> always build error

test in local, find that the grpc-core [maven-metadata.xml](https://repo.maven.apache.org/maven2/io/grpc/grpc-core/maven-metadata.xml) file does not contains version 1.28.0, seems so strange!

image

**Comment by z7658329** [comment]
> 

upgrade grpc-core to 1.28.1, then build success ! 
image

**Review by imbajin — APPROVED** [review]
Good, and you could request review here

image

**Review by javeme — APPROVED** [review]

**Pull request merged**